elk/training/preprocessing.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+import logging
+import torch
+
+
+def normalize(data: torch.Tensor):
```

Comment:
that's not CCS' normalization
(see my PR on normalization)

Reply:
It's the normalization from the paper, but yes we should include multiple options